### PR TITLE
Add gvlsplit (personal split kb)

### PR DESCRIPTION
**Pull request by gvlatko**
## Description

I am trying to build a custom split handwired keyboard that i've designed.
I've read that i have to make a PR for my layout to appear in the QMK configurator.
I've no idea what i am doing atm or if this PR is proper.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [ ] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/gvlsplit/config.h [hunk 1]
```diff
@@ -0,0 +1,20 @@
+// Copyright 2023 Vlatko Gjurgjinoski (@Vlatko Gjurgjinoski)
```
Comment (fauxpark) [inline keyboards/gvlsplit/config.h:1]: This file can be deleted.

File: keyboards/gvlsplit/readme.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# gvlsplit
+
+![gvlsplit](imgur.com image replace me!)
+
+*A short description of the keyboard/project*
+
+* Keyboard Maintainer: [Vlatko Gjurgjinoski](https://github.com/Vlatko Gjurgjinoski)
+* Hardware Supported: *The PCBs, controllers supported*
+* Hardware Availability: *Links to where you can find this hardware*
```
Comment (fauxpark) [inline keyboards/gvlsplit/readme.md:9]: Please fill these out.

File: keyboards/gvlsplit/info.json [hunk 3]
```diff
@@ -0,0 +1,424 @@
+{
```
Comment (fauxpark) [inline keyboards/gvlsplit/info.json:1]: Please run `qmk format-json` on this file.

File: keyboards/gvlsplit/config.h (hunk 1 above)
Comment (gvlatko) [reply keyboards/gvlsplit/config.h]: Deleted

File: keyboards/gvlsplit/readme.md (hunk 2 above)
Comment (gvlatko) [reply keyboards/gvlsplit/readme.md]: updated

File: keyboards/gvlsplit/info.json (hunk 3 above)
Comment (gvlatko) [reply keyboards/gvlsplit/info.json:1]: ran `qmk format-json -f keymap info.json`

Comment (fauxpark) [reply keyboards/gvlsplit/info.json:1]: This isn't a keymap.

File: keyboards/gvlsplit/readme.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
+# gvlsplit
+
+![gvlsplit](https://imgur.com/a/SSFVDgD)
+
+A personal custom split handwired keyboard with 3d printed case.
+
+-   Keyboard Maintainer: [Vlatko Gjurgjinoski](https://github.com/Vlatko Gjurgjinoski)
```
Comment (jasonhazel) [inline keyboards/gvlsplit/readme.md:7]:
```suggestion
-   Keyboard Maintainer: [Vlatko Gjurgjinoski](https://github.com/gvlatko)
```

**Review by zvecr — CHANGES_REQUESTED** [review]

File: keyboards/gvlsplit/readme.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
+# gvlsplit
+
+![gvlsplit](https://imgur.com/a/SSFVDgD)
+
+A personal custom split handwired keyboard with 3d printed case.
```
[the remark this answers is not in the document]
Comment (gvlatko) [reply keyboards/gvlsplit/readme.md:5]: moved to handwired

File: keyboards/handwired/gvlsplit/readme.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
+# gvlsplit
+
+![gvlsplit](https://imgur.com/a/SSFVDgD)
+
+A personal custom split handwired keyboard with 3d printed case.
+
+-   Keyboard Maintainer: [Vlatko Gjurgjinoski](https://github.com/Vlatko Gjurgjinoski)
+-   Hardware Supported: Promicro atmeta32u4 x2, TRRS ports x2
+-   Hardware Availability: https://aliexpress.com
```
Comment (fauxpark) [inline keyboards/handwired/gvlsplit/readme.md:9]:
```suggestion
* Keyboard Maintainer: [Vlatko Gjurgjinoski](https://github.com/gvlatko)
* Hardware Supported: Pro Micro
* Hardware Availability: https://aliexpress.com
```

File: keyboards/handwired/gvlsplit/readme.md [hunk 7]
```diff
@@ -0,0 +1,27 @@
+# gvlsplit
+
+![gvlsplit](https://imgur.com/a/SSFVDgD)
```
Comment (fauxpark) [inline keyboards/handwired/gvlsplit/readme.md:3]:
```suggestion
![gvlsplit](https://i.imgur.com/X9vh2FI.png)
```

File: keyboards/handwired/gvlsplit/keymaps/default/keymap.c [hunk 8]
```diff
@@ -0,0 +1,49 @@
+// Copyright 2023 QMK
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include QMK_KEYBOARD_H
+
+#define BASE_LAYER 0
+#define RAISE_LAYER 1
+#define LOWER_LAYER 2
+#define FN_LAYER 3
```
Comment (fauxpark) [inline keyboards/handwired/gvlsplit/keymaps/default/keymap.c:9]:
```suggestion
enum layer_names {
    _BASE,
    _RAISE,
    _LOWER,
    _FN
};
```

File: keyboards/handwired/gvlsplit/keymaps/default/keymap.c [hunk 9]
```diff
@@ -0,0 +1,49 @@
...
+        KC_UP, KC_LCTL, KC_LGUI, KC_LALT,
+        TO(1), KC_SPC,
+        KC_BSPC,
+        KC_ESC, KC_ENT, TG(2), MO(3),
+        KC_LEFT,
+        KC_DOWN,
+        KC_RGHT
+    ),
+
+    [RAISE_LAYER] = LAYOUT(
```
Comment (fauxpark) [inline keyboards/handwired/gvlsplit/keymaps/default/keymap.c:26]:
```suggestion
    [_RAISE] = LAYOUT(
```

File: keyboards/handwired/gvlsplit/keymaps/default/keymap.c [hunk 10]
```diff
@@ -0,0 +1,49 @@
...
+        KC_TILD, KC_EXLM, KC_HASH, KC_DLR, KC_PERC, KC_CIRC, KC_AMPR, KC_ASTR, KC_LPRN, KC_RPRN, KC_NO, KC_NO, KC_NO,
+        KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_9, KC_0, KC_LBRC, KC_RBRC, KC_NO, KC_NO, KC_NO,
+        KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_LEFT, KC_DOWN, KC_UP, KC_RGHT, KC_NO, KC_NO, KC_NO,
+        KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO,
+        KC_NO, KC_NO, KC_NO, KC_TRNS, KC_NO, KC_NO, KC_DEL, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO
+    ),
+
+    [LOWER_LAYER] = LAYOUT(
```
Comment (fauxpark) [inline keyboards/handwired/gvlsplit/keymaps/default/keymap.c:34]:
```suggestion
    [_LOWER] = LAYOUT(
```

File: keyboards/handwired/gvlsplit/keymaps/default/keymap.c [hunk 11]
```diff
@@ -0,0 +1,49 @@
...
+        KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO,
+        KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_HOME, KC_END, KC_PGUP, KC_PGDN, KC_NO, KC_NO, KC_NO,
+        KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_LEFT, KC_DOWN, KC_UP, KC_RGHT, KC_NO, KC_NO, KC_NO,
+        KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO,
+        KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_DEL, KC_NO, KC_NO, KC_TRNS, KC_NO, KC_NO, KC_NO, KC_NO
+    ),
+
+    [FN_LAYER] = LAYOUT(
```
Comment (fauxpark) [inline keyboards/handwired/gvlsplit/keymaps/default/keymap.c:42]:
```suggestion
    [_FN] = LAYOUT(
```

File: keyboards/gvlsplit/rules.mk [hunk 12]
```diff
@@ -0,0 +1 @@
+SPLIT_KEYBOARD = yes
```
Comment (fauxpark) [inline keyboards/gvlsplit/rules.mk:1]: These files can be deleted.

File: keyboards/handwired/gvlsplit/info.json [hunk 13]
```diff
@@ -0,0 +1,424 @@
...
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "keyboard_name": "gvlsplit",
+    "layouts": {
+        "LAYOUT": {
+            "layout": [
+                {
```
Comment (fauxpark) [inline keyboards/handwired/gvlsplit/info.json:16]: You still haven't formatted this file properly.

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/handwired/gvlsplit/info.json [hunk 14]
```diff
@@ -0,0 +1,97 @@
...
+    },
+    "processor": "atmega32u4",
+    "url": "",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x0000",
+        "vid": "0xFEED"
+    },
+    "layouts": {
+        "LAYOUT_gvlsplit": {
```
Comment (fauxpark) [inline keyboards/handwired/gvlsplit/info.json:27]:
```suggestion
        "LAYOUT": {
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/handwired/gvlsplit/readme.md [hunk 15]
```diff
@@ -0,0 +1,27 @@
+# gvlsplit
+
+![gvlsplit](https://i.imgur.com/X9vh2FI.png)
```
Comment (drashna) [inline keyboards/handwired/gvlsplit/readme.md:3]:
```suggestion
![gvlsplit](https://i.imgur.com/X9vh2FIh.png)
```

File: keyboards/handwired/gvlsplit/keymaps/default/keymap.c [hunk 16]
```diff
@@ -0,0 +1,51 @@
...
+        KC_TAB, KC_Q, KC_W, KC_E, KC_R, KC_T, KC_Y, KC_U, KC_I, KC_O, KC_P, KC_LBRC, KC_RBRC,
+        KC_CAPS, KC_A, KC_S, KC_D, KC_F, KC_G, KC_H, KC_J, KC_K, KC_L, KC_SCLN, KC_QUOT, KC_BSLS,
+        KC_LSFT, KC_Z, KC_X, KC_C, KC_V, KC_B, KC_N, KC_M, KC_COMM, KC_DOT, KC_SLSH,
+        KC_UP, KC_LCTL, KC_LGUI, KC_LALT,
+        TO(1), KC_SPC,
+        KC_BSPC,
+        KC_ESC, KC_ENT, TG(2), MO(3),
+        KC_LEFT,
+        KC_DOWN,
+        KC_RGHT
```
Comment (drashna) [inline keyboards/handwired/gvlsplit/keymaps/default/keymap.c:25]:
```suggestion
        KC_UP, KC_LCTL, KC_LGUI, KC_LALT, TO(1), KC_SPC,  KC_BSPC, KC_ESC, KC_ENT, TG(2), MO(3), KC_LEFT, KC_DOWN, KC_RGHT
```

File: keyboards/handwired/gvlsplit/readme.md (hunk 15 above)
Comment (gvlatko) [reply keyboards/handwired/gvlsplit/readme.md:3]: whats the difference here? was the previous url not working? both urls load the image for me

**Review by drashna — APPROVED** [review]

**Review by drashna — APPROVED** [review]

**Pull request closed without merging**